packages/longhorn/longhorn-1.2/package.yaml

```diff
@@ -1,7 +1,7 @@
 url: https://github.com/longhorn/charts.git
 subdirectory: charts/longhorn
-commit: d699089b1a62011c9cdeb6e8dacba203f30655a8
-version: 100.1.2
+commit: d3ac3c2ee47cd9bd420b36cdef0d4e4fb24f9622
+version: 100.1.3
```

Comment:
Looks like the longhorn chart version we last released is `100.1.1+up1.2.3` for the 1.2 version line, so you don't need to bump this again if it was bumped already. The next to-be-released version should be one step up from the previous, so that would be `100.1.2+up1.2.4`. You need to remove the entries in release.yaml and assets associated with `100.1.2+up1.2.3` and `100.1.3+up1.2.4`, and generate new ones using `100.1.2` instead to get `100.1.2+up1.2.4`.

Reply:
It seems there is an intermediate change (https://github.com/rancher/charts/commit/41c2d8dd4fde86e477f877fc0df57fb865197130) to cause another `100.1.2+up1.2.3`. Fixed.